### ISSUE TITLE
Support macos pkg with jpackage

### ISSUE THREAD

**Issue by jvalkeal** [spring-projects/spring-cli#121]
We currently have `msi` for windows, `rpm`/`deb` for linux. There was initial restrictions on macos `pkg` package so that if fails if package version is below `1.0.0`. Should look if we need to wait or if there's some workarounds.

**Comment by jvalkeal** [comment]
Yes things fail on jreleaser level. I think it's check in jreleaser itself to fail fast as `pkg` install on macos would always fail:
```
Error:  The first number in an app-version cannot be zero or negative (0.8.0).
```

**Comment by aalmiray** [comment]
This is a constraint violation that occurs when invoking the `jpackage` command itself. You may override the computed value by providing an explicit value for the `appVersion` property in the `macos` section of the `jpackage` assembler.

For some odd reason the number must conform to semver (without tags) and the major version must be greater than `0`.

**Comment by jvalkeal** [comment]
Ah yeah I now remember seeing an openjdk ticket adding fast failure. Thanks for a tip to forcing the version. Gotta love apple's logic on these things.
